util/packaging/homebrew/chapel-release.rb

```diff
@@ -50,6 +44,8 @@ def install
     (libexec/"chplconfig").write <<~EOS
       CHPL_RE2=bundled
       CHPL_GMP=system
+      CHPL_MEM=cstdlib
+      CHPL_TASKS=fifo
```

Comment:
Why are we using cstdlib/fifo here? I don't see a comment explaining it here or in the PR message. I would expect it is a workaround. Is it an old workaround that is no longer needed?

Reply:
Here was the change that introduced those lines:   https://github.com/chapel-lang/chapel/pull/23415.  We have not tested without these lines but I can remove them and give the formula test.

Reply:
I will update this to add comments but we need those two lines or the homebrew build fails.   Not sure what the work around is at this point beyond those two lines.

Reply:
Comments added but will need to raise an issue for fixing this...